src/app/Caches/CacheMetrics.tsx

```diff
@@ -216,19 +221,25 @@ const CacheMetrics = (props: { cacheName: string; display: boolean }) => {
         <CardBody>
           <TextContent>
             <TextList component={TextListVariants.dl}>
-              <TextListItem component={TextListItemVariants.dt}>
+              <TextListItem aria-label="view-cache-metrics-off-heap" component={TextListItemVariants.dt}>
                 {displayUtils.formatNumber(
-                  stats.current_number_of_entries_in_memory
+                  stats.off_heap_memory_used
                 )}
               </TextListItem>
               <TextListItem component={TextListItemVariants.dd}>
-                Current number
+                <MoreInfoTooltip
+                  label={t('caches.cache-metrics.cache-size-off-heap')}
+                  toolTip={'Choose \'Custom Type\' to specify keys with custom `_type` json value type.'}
+                />
               </TextListItem>
-              <TextListItem component={TextListItemVariants.dt}>
+              <TextListItem aria-label="view-cache-metrics-heap" component={TextListItemVariants.dt}>
                 {displayUtils.formatNumber(stats.data_memory_used)}
               </TextListItem>
               <TextListItem component={TextListItemVariants.dd}>
-                Total in use
+                <MoreInfoTooltip
+                  label={t('caches.cache-metrics.cache-size-heap')}
+                  toolTip={'Choose \'Custom Type\' to specify keys with custom `_type` json value type.'}
```

Comment:
Change "Choose \'Custom Type\' to specify keys with custom `_type` json value type." to

"Displays the amount of JVM heap memory that data in the cache uses. Total memory usage is higher because it includes internal data structures."